driver/src/bin/main.rs

```diff
@@ -16,7 +16,8 @@ fn main() {
     let db_instance = MongoDB::new(db_host, db_port).unwrap();
     let contract = SnappContractImpl::new().unwrap();
 
-    let solver_env_var = env::var("LINEAR_OPTIMIZATION_SOLVER").unwrap_or("0".to_string());
+    let solver_env_var = env::var("LINEAR_OPTIMIZATION_SOLVER")
+        .unwrap_or_else(|_| "0".to_string());
```

Comment:
Intuitively, I'd assume that using a closure here is less efficient than the constant we used to use before. Do you know why this version is supposed to be better?

Reply:
found it: https://rust-lang.github.io/rust-clippy/master/index.html#or_fun_call

Reply:
I can't tell if this link is for or against the change proposed here. Looks like an argument for the lint warning.